### PR TITLE
Fix snackbar logic and implement exception handler for program crashes

### PR DESCRIPTION
**Pull request by RohanSeam**
# Summary
Update front-end snackbar logic to add negative case. Also, implemented new module for exception/error handling to handle potential program crashes.

## What I did here
- Added new module and updated settings routes.

## How to test
- Can't, currently unlit :D

### CODE REVIEW

File: application/templates/settings.html [hunk 1]
```diff
@@ -131,6 +131,16 @@ <h1 class="h1">System Preferences</h1>
     <div id="snackbar">Update successful :O)</div>
 
     <script>
+      // Allows us to format strings in a pythonic way.
+      String.prototype.format = function() {
+        var formatted = this;
+        for (var i = 0; i < arguments.length; i++) {
+            var regexp = new RegExp('\\{'+i+'\\}', 'gi');
```
Comment (CurranSeam) [inline application/templates/settings.html:138]: 😮

File: application/utils/exception_handler.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+import inspect
+import logging
+import sys
+import traceback
+
+# Might move in future. Custom exception for bad function call.
+class NonCallableFunctionException(Exception):
+    pass
+
+def try_exec(func, *args):
```
Comment (CurranSeam) [inline application/utils/exception_handler.py:13]: Nice!

File: application/utils/exception_handler.py [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    """
+    Executes the passed function within a try-except block.
+    If an exception occurs, the error message is handled and 
+    logged to output files for debugging.
+    Ex: try_exec(sms_service.send_message, list(CAMERAS.keys())[idx], current_time, FEED_URL, filepath)
+
+    :param func: function to execute within try-except.
+    :param *args: arguments to pass when executing func.
+    :return: a tuple with return code and results from func.
+    """
```
Comment (CurranSeam) [inline application/utils/exception_handler.py:23]: good comment! Will be super useful :D

Comment (CurranSeam) [reply application/utils/exception_handler.py:23]: Might want to change the function in the example later on 😄

File: driver.py [hunk 4]
```diff
@@ -45,6 +49,20 @@ def run(self):
...
+    handler = TimedRotatingFileHandler(filename="logging/kernel_debug.log",
+                                       when="d",
+                                       interval=1,
+                                       backupCount=BACKUP_FILE_COUNT)
+
+    fmt = '%(asctime)s %(levelname)s %(message)s'
+    formatter = logging.Formatter(fmt=fmt, datefmt='%m/%d/%Y %H:%M:%S')
+    handler.setFormatter(formatter)
+    logger.addHandler(handler)
+
```
Comment (CurranSeam) [inline driver.py:65]: Lit :D

**Review by CurranSeam — APPROVED** [review]
We can finally now debug! 🚀

File: application/utils/exception_handler.py (hunk 3 above)
Comment (RohanSeam) [reply application/utils/exception_handler.py:23]: Facts!

**Pull request merged**